modules/govuk/manifests/apps/email_alert_api/checks.pp

```diff
@@ -4,8 +4,20 @@
 #
 class govuk::apps::email_alert_api::checks(
   $ensure = present,
+  $queue_resources = {
+    'delivery_immediate_high'     => { },
+    'delivery_immediate'          => { },
+    'delivery_digest'             => { },
+    'email_generation_immediate'  => { },
+    'process_and_generate_emails' => { },
+    'default'                     => { },
+    'email_generation_digest'     => { },
+    'cleanup'                     => { },
+  },
 ) {
 
+  create_resources(sidekiq_queue_checks, $queue_resources)
```

Comment:
I think using a resource declaration with an array of titles [1] might be a neater way of expressing this, providing you don't need to vary the `queue_resources` parameter value.

1: https://puppet.com/docs/puppet/3.8/lang_resources.html#condensed-forms